File: solutions/go/pangram/1/pangram.go
package pangram

import "strings"

func IsPangram(input string) bool {
	upper := strings.ToUpper(input)

	for _, char := range("ABCDEFGHIJKLMNOPQRSTUVWXYZ") {
		if !strings.ContainsRune(upper, char) {
			return false
		}
	}
	return true
}

File: solutions/javascript/pangram/1/pangram.js
var Pangram = function(phrase) {
  this.phrase = phrase
}
Pangram.prototype.isPangram = function() {
  alphabet = 'abcdefghijklmnopqrstuvwxyz'
  for(var i = 0; i < this.phrase.length; i++) {
    alphabet = alphabet.replace(this.phrase[i].toLowerCase(),'')
  }
  pangram = alphabet.length === 0 ? true : false
  return pangram
}
module.exports = Pangram